### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by edras**
included additional information about urls.py

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -104,11 +102,22 @@ A simple Line Chart example.
                     [87, 21, 94, 3, 90, 13, 65]]
     
     
-    line_chart = TemplateView.as_view(template_name='line_chart.html')
-    line_chart_json = LineChartJSONView.as_view()
```
Comment (Natim) [inline README.rst:108]: Maybe we can keep this and then import them in the urls.py file?

File: README.rst [hunk 2]
```diff
@@ -104,11 +102,22 @@ A simple Line Chart example.
...
+.. code-block:: python
+
+    from django.views.generic import TemplateView
+    from .views import LineChartJSONView
+    
+    urlpatterns = [
+      '...',
+      path('chart', TemplateView.as_view(template_name='chart.html'), name='line_chart'),
+      path('chartJSON', views.LineChartJSONView.as_view(), name='line_chart_json'),
+    ]
```
Comment (Natim) [inline README.rst:117]:
```suggestion
    from django.views.generic import TemplateView
    from .views import line_chart, line_chart_json
    
    urlpatterns = [
      '...',
      path('chart', line_chart),
      path('chartJSON', line_chart_json),
    ]
```

**Comment by Natim** [comment]
Thank you for starting this PR.

I think that generally it is best to define views in the views.py file and then import the view in the urls.py file.

It makes things more readable. What do you think?

**Comment by edras** [comment]
I am following what is described in django 3.0 documentation
https://docs.djangoproject.com/en/3.0/topics/class-based-views/

In my projects, I import the view and call the classes as needed in my
urls.py
But I am Ok in keeping things readable.

Best regards,

Edras R. Pacola
edras.pacola@gmail.com
Engenheiro Eletrônico

"Engenheiro: agente de transformações tecnológicas e sociais."


Am Di., 5. Mai 2020 um 13:22 Uhr schrieb Rémy HUBSCHER <
notifications@github.com>:

**Pull request merged**